tests/security/test_blobstorage.py

```diff
@@ -27,7 +28,8 @@ def test_encoded_format(config, sample_input, encrypt):
 
 
 @hypothesis.given(s.text(), s.booleans())
-def test_message_body_storage(config, message, sample_input, encrypt):
+def test_message_body_storage(config, sample_input, encrypt):
+    message = Message()
```

Comment:
```hypothesis.errors.FailedHealthCheck: security/test_blobstorage.py::test_message_body_storage uses the 'message' fixture, which is reset between function calls but not between test cases generated by `@given(...)`.  You can change it to a module- or session-scoped fixture if it is safe to reuse; if not we recommend using a context manager inside your test function.  See https://docs.pytest.org/en/latest/fixture.html#sharing-test-data for details on fixture scope.```

The fixture did more things than needed for this test anyway. so I just replaced it with constructor...